config.go

```diff
@@ -43,52 +53,52 @@ tools:
-	Tags         TagRegistrationConfig `json:"tags" yaml:"tags"`
-	Tools        []string              `json:"tools" yaml:"tools"`               // JQ expressions that return a single map[string]string or a []map[string]string
-	Repositories []string              `json:"repositories" yaml:"repositories"` // JQ expressions that return a single string or []string or map[string]string or a []map[string]string
+	Aliases      []string                   `json:"aliases" yaml:"aliases"` // JQ expressions that return a single string or a []string
+	Description  string                     `json:"description" yaml:"description"`
+	Framework    string                     `json:"framework" yaml:"framework"`
+	Language     string                     `json:"language" yaml:"language"`
+	Lifecycle    string                     `json:"lifecycle" yaml:"lifecycle"`
+	Name         string                     `json:"name" yaml:"name"`
+	Owner        string                     `json:"owner" yaml:"owner"`
+	Product      string                     `json:"product" yaml:"product"`
+	Properties   PropertyRegistrationConfig `json:"properties" yaml:"properties"`
+	Repositories []string                   `json:"repositories" yaml:"repositories"` // JQ expressions that return a single string or []string or map[string]string or a []map[string]string
+	System       string                     `json:"system" yaml:"system"`
+	Tags         TagRegistrationConfig      `json:"tags" yaml:"tags"`
+	Tier         string                     `json:"tier" yaml:"tier"`
+	Tools        []string                   `json:"tools" yaml:"tools"` // JQ expressions that return a single map[string]string or a []map[string]string
 }
 
 // ServiceRegistration represents the parsed json data from a ServiceRegistrationConfig
 type ServiceRegistration struct {
-	Name         string                                  `json:",omitempty"`
+	Aliases      []string                                `json:",omitempty"`
 	Description  string                                  `json:",omitempty"`
-	Owner        string                                  `json:",omitempty"`
+	Framework    string                                  `json:",omitempty"`
+	Language     string                                  `json:",omitempty"`
 	Lifecycle    string                                  `json:",omitempty"`
-	Tier         string                                  `json:",omitempty"`
+	Name         string                                  `json:",omitempty"`
+	Owner        string                                  `json:",omitempty"`
 	Product      string                                  `json:",omitempty"`
-	Language     string                                  `json:",omitempty"`
-	Framework    string                                  `json:",omitempty"`
+	Properties   []opslevel.PropertyInput                `json:",omitempty"`
```

Comment:
This should likely just return a `map[sting]opslevel.JsonString`

Reply:
Changed this to do exactly that